ext-src/swoole_http_response.cc

```diff
@@ -1042,6 +1042,11 @@ static void php_swoole_http_response_cookie(INTERNAL_FUNCTION_PARAMETERS, const
             zval max_age;
             ZVAL_DOUBLE(&max_age, diff);
             convert_to_string(&max_age);
+            cookie_size += sizeof(max_age);
+            char *new_cookie = (char *) emalloc(cookie_size);
+            strlcpy(new_cookie, cookie, cookie_size);
+            efree(cookie);
+            cookie = new_cookie;
```

Comment:
```c++
size_t cookie_size = name_len /* + value_len */ + path_len + domain_len + 200 + samesite_len + priority_len;
```
Attempting to modify the allocated memory size here.

Reply:
We don't know the exact length of `max_age` until `expires` is processed, though. 🧐

Reply:
@NathanFreeman, I've updated the code to try and calculate the cookie_size. How is it now?

Reply:
It would be better to use `sizeof("") - 1` to represent the length